src/transforms/lua/v2/interop/util.rs

```diff
@@ -35,7 +33,8 @@ pub fn table_to_timestamp<'a>(t: LuaTable<'a>) -> LuaResult<DateTime<Utc>> {
     let hour = t.get("hour")?;
     let min = t.get("min")?;
     let sec = t.get("sec")?;
-    Ok(Utc.ymd(year, month, day).and_hms(hour, min, sec))
+    let nano = t.get("nanosecond").unwrap_or(0);
```

Comment:
I'm a rust newbie, not sure what the appropriate idiom is here.

Reply:
I think it is better to use something like `t.get::<_, Option<u64>>("nanosecond")?.unwrap_or(0)`. The difference is that if a `Result` is not `Ok(...)`, then the error might be caused by reasons other than missing key, such as runtime errors. On the other hand, if there are no runtime errors, but the key is missing, then `get::<_, Option<u64>>?` would result in `None` and then applying `unwrap_or(..)` on an `Option`, as opposed to a result, would not mean ignoring any errors.

Also see https://github.com/timberio/vector/issues/2332#issuecomment-614047567 about the name of the field.

Reply:
Done, thanks for explaining.